packages/nextcloud/lib/ids.dart

```diff
@@ -0,0 +1,25 @@
+// ignore_for_file: public_member_api_docs
+
+sealed class AppIDs {
```

Comment:
An enhanced enum is better.

Reply:
As I said, this is not possible due to the need have the IDs as const.

Reply:
Makes sense.

Then please make the class final. There is no reason to have it sealed here.

Reply:
If it complains about it being a utility class just add a hidden constructor.